client/src/pages/home/HomePage.js

```diff
@@ -1,19 +1,61 @@
-import React from "react";
+import React, { useState } from "react";
 import ProverbList from "./../proverb/ProverbList";
-
+import AddProverb from "./../proverb/AddProverb";
 import Section from "./../../components/Section";
-import Breadcrumb from './../../components/Breadcrumb'
+import Breadcrumb from "./../../components/Breadcrumb";
+import Modal from "./../../components/Modal";
+import Button from "./../../components/Button";
+
+import './HomePage.css'
 
 const HomePage = () => {
+  //Modal States
+  const [modal, setModal] = useState({
+    isOpen: false,
+    type: "",
+    id: "",
+  });
+
+  //Modal Handlers
+
+  const handleShow = (type) => async (e) => {
```

Comment:
```suggestion
  const handleShowModal = (type) => async (e) => {
```

Let's make this function name even more specific so it is very clear what it is for